README.md

```diff
@@ -66,6 +66,7 @@ The judge can also grade in the languages listed below:
 * Groovy
 * Haskell
 * INTERCAL
+* JavaScript (Node.js or V8)
```

Comment:
For consistency with C++ above.

```suggestion
* JavaScript (Node.js and V8)
```

Reply:
Changed it to "JavaScript (Node.js and V8)"